samples/Basic_Blink/component.mk

```diff
@@ -29,11 +29,16 @@
 # COMPONENT_CFLAGS :=
 # COMPONENT_CXXFLAGS :=
 
-## Configure flash parameters (for ESP12-E and other new boards):
-# SPI_MODE := dio
+## Configure hardware
+# Default is 'standard' (no spiffs), can also provide your own
+#HWCONFIG := spiffs
 
-## SPIFFS options
+## SPIFFS options (deprecated: use HWCONFIG)
+##	0: (default) use `HWCONFIG=standard` as default
```

Comment:
The documentation does not match the meaning of the values. For example DISABLE_SPIFFS=1 is `HWCONFIG=standard`.

Reply:
Thanks. I'll fix this.

Reply:
Should be something like this:

```
## SPIFFS filesystem options (requires HWCONFIG=spiffs)
## 0: (default) Build filesystem image from content of 'files' directory
## 1: Leave spiffs partition un-initialised
```

Reply:
Removed entirely